src/main/java/de/retest/recheck/ignore/SearchFilterFiles.java

```diff
@@ -15,6 +26,14 @@
 
 	private SearchFilterFiles() {}
 
+	private static Path resolveFilterPath() {
```

Comment:
This method duplicates code from `ProjectConfiguration.ensureProjectConfigurationInitialized()`, please refactore.